src/panels/config/info/ha-config-info.ts

```diff
@@ -165,13 +165,17 @@ class HaConfigInfo extends LitElement {
                 `
               )}
             </mwc-list>
-            <p class="config-path">
-              ${this.hass.localize(
-                "ui.panel.config.info.path_configuration",
-                "path",
-                hass.config.config_dir
-              )}
-            </p>
+            ${hass.config.config_dir !== "/config"
```

Comment:
It's the default place and not really needed unless its somewhere weird.

Reply:
Right, so it was originally added there to have a place to point to where users could see which path is used (as it can be different on different installation methods).

Removing is only for (basically OS & Supervised, when using default) is a bit inconsistent, as it would sometime be here, sometime it won't. This also means we can't point to it reliably anymore.

That is fine, but in that case, maybe just remove it altogether (and put it somewhere else in a consistent way).

Reply:
My main goal for this is if its `/config` then its hidden, we know its `/config` but also its not something users need to see if they aren't using YAML files.

I would rather not show this to basic users. Would you be more open to hiding if non advanced and the location is `/config`?

Reply:
> I would rather not show this to basic users. 

The whole goal why this was added, **is** for newer user. More advanced and longer time users will know where their config is.

This was added to have "somewhere" to point people too when they don't know.

I think you goal of removing conflicts with the goal why it was added in the first place; and causes inconsistency on when its show or not.

Reply:
Sure. I get the original reason but I think the basic user definition has changed since then. If you don't have advanced mode turned on then you probably are not messing with configuration files.

It's not that it is ugly. It more the goal that some users don't need or want to know about configuration files or advanced features as file editing. They just need to know about their settings menu and settings in the UI. 

Im fine keeping it as is really. It was more just moving more away from the basic user's eyes if they don't need to see it. 🤷

Reply:
> Sure. I get the original reason but I think the basic user definition has changed since then. If you don't have advanced mode turned on then you probably are not messing with configuration files.

I'm not fighting that. If we think it's fine to remove, remove it. Keep it if needed, or move it.

My concern that I'm fighting is inconsistency. As you no longer can point people to here to say: "Look there is your config path".... as well... depends... what your config path is or else, there is just nothing.

Reply:
Night of sleep: How about we show this on the system health page and remove it from here altogether?

Reply:
I like that and it fits in there better anyway